src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -112,49 +120,48 @@ def run_conversion(
         es_key: str = None,
         compression: Optional[str] = None,
         compression_opts: Optional[int] = None,
-        iterator_type: Optional[str] = "v2",
+        iterator_type: str = "v2",
```

Comment:
Same as above

Reply:
is this true every time `iterator_type` is used? It's hard for me to tell since this was not indicated in the docstrings in the first place

Reply:
It is true for the ecephys and ophys pipelines that rely on the `spikeinterface.py` and `roiextractors.py` modules. So, this should be true here as well (this uses the `write_recording` function).

Btw, looking at this specific module it seems that we are duplicating `run_conversion`. The function here is identical to that of the parent class `BaseRecordingExtractorInterface`.